database_controller.py

```diff
@@ -56,11 +56,15 @@ def get_measurement_items(self):
 
 
     def get_cages_in_group(self, group):
-        return (self.cages_in_group[group])
+        if group in self.cages_in_group.keys():
```

Comment:
In the future, utilize the .get() function instead of accessing the dictionary two times.